baseplate/context/memcache/__init__.py

```diff
@@ -107,31 +107,153 @@ def __init__(self, context_name, server_span, pooled_client):
+
+    def gets_many(self, keys):
+        with self._make_span("gets_many") as span:
+            span.set_tag("key_count", len(keys))
+            return self.pooled_client.gets_many(keys)
+
+    def delete(self, key, noreply=None):
+        with self._make_span("delete") as span:
+            span.set_tag("key", key)
+            span.set_tag("noreply", noreply)
+            return self.pooled_client.delete(key, noreply=noreply)
+
+    def delete_many(self, keys, noreply=None):
+        with self._make_span("delete_many") as span:
+            span.set_tag("key_count", len(keys))
+            span.set_tag("noreply", noreply)
+            return self.pooled_client.delete_many(keys, noreply=noreply)
+
+    def add(self, key, value, expire=0, noreply=None):
+        with self._make_span("add") as span:
+            span.set_tag("key", key)
+            span.set_tag("value", value)
+            span.set_tag("expire", expire)
+            span.set_tag("noreply", noreply)
+            return self.pooled_client.add(key, value, expire=expire, noreply=noreply)
+
+    def incr(self, key, value, noreply=False):
+        with self._make_span("incr") as span:
+            span.set_tag("key", key)
+            span.set_tag("value", value)
+            span.set_tag("noreply", noreply)
+            return self.pooled_client.incr(key, value, noreply=noreply)
+
+    def decr(self, key, value, noreply=False):
+        with self._make_span("decr") as span:
+            span.set_tag("key", key)
+            span.set_tag("value", value)
+            span.set_tag("noreply", noreply)
+            return self.pooled_client.decr(key, value, noreply=noreply)
+
+    def touch(self, key, expire=0, noreply=None):
+        with self._make_span("touch") as span:
+            span.set_tag("key", key)
+            span.set_tag("expire", expire)
+            span.set_tag("noreply", noreply)
+            return self.pooled_client.touch(key, expire=expire, noreply=noreply)
+
+    def stats(self, *args):
+        with self._make_span("stats"):
+            return self.pooled_client.stats(*args)
+
+    def flush_all(self, delay=0, noreply=None):
+        with self._make_span("flush_all") as span:
+            span.set_tag("delay", delay)
+            span.set_tag("noreply", noreply)
+            return self.pooled_client.flush_all(delay=delay, noreply=noreply)
+
+    def quit(self):
+        with self._make_span("quit"):
+            return self.pooled_client.quit()
+
+    def _make_span(self, method_name):
+        """Get a child span of the current server span.
+
+        The returned span is tagged with ``method_name`` and given a name
+        that corresponds to the current context name and called method.
+
+        """
+        trace_name = "{}.{}".format(self.context_name, method_name)
+        span = self.server_span.make_child(trace_name)
+        span.set_tag("method", method_name)
+        return span
+
+
+def make_keys_str(keys):
```

Comment:
i don't see this function getting used anywhere? are the `key_cnt` tags supposed to be updated to use it?